artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverter.java

```diff
@@ -216,8 +216,8 @@ public static org.apache.activemq.artemis.api.core.Message inbound(final Message
       }
 
       final Object scheduledDelay = messageSend.getProperties().get(ScheduledMessage.AMQ_SCHEDULED_DELAY);
-      if (scheduledDelay instanceof Long) {
-         coreMessage.putLongProperty(org.apache.activemq.artemis.api.core.Message.HDR_SCHEDULED_DELIVERY_TIME, System.currentTimeMillis() + ((Long) scheduledDelay));
+      if (scheduledDelay instanceof Long long1) {
```

Comment:
Something like delayMs or delayLong would be far more readable later.

Just about every name added below is equally unhelpful, and results in less readable code than the original.